arborist/user.go

```diff
@@ -150,6 +150,56 @@ func userWithName(db *sqlx.DB, name string) (*UserFromQuery, error) {
 	return &user, nil
 }
 
+type PolicyInfoFromQuery struct {
+	Name           string `db:"name"`
+	Authz_provider string `db:"authz_provider"`
+	Type           string `db:"type"`
+}
+
+func fetchPolicyInfo(db *sqlx.DB, user_name string, policy_name string) (*PolicyInfoFromQuery, error) {
```

Comment:
i was confused at first why this function was in the `user` module, maybe rename it `fetchUserPolicyInfo`?